src/axolotl/utils/models.py

```diff
@@ -535,7 +536,7 @@ def load_model(
 
     model, lora_config = load_adapter(model, cfg, cfg.adapter)
 
-    if cfg.ddp and not load_in_8bit:
+    if cfg.ddp and not load_in_8bit and not load_in_4bit:
         model.to(f"cuda:{cfg.local_rank}")
```

Comment:
what's the side effect of this change? Does it mean, the models will all live on gpu0 for 4 bit now?